src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,102 @@
+package mate.jdbc.dao;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String updateQuery = "INSERT INTO manufacturers(name, country) values (?, ?);";
+        try (PreparedStatement manufacturerStatement = ConnectionUtil.getConnection()
+                .prepareStatement(updateQuery, Statement.RETURN_GENERATED_KEYS)) {
+            manufacturerStatement.setString(1, manufacturer.getName());
+            manufacturerStatement.setString(2, manufacturer.getCountry());
+            manufacturerStatement.executeUpdate();
+            ResultSet generatedKeys = manufacturerStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can`t create manufacturer in DB", throwables);
```

Comment:
```suggestion
            throw new DataProcessingException("Can`t create manufacturer in DB " + manufacturer, throwables);
```
add more information